gha/expired-branches.sh

```diff
@@ -0,0 +1,66 @@
+# define expiration date
```

Comment:
You want a hash bang up in here :) 

`#! /usr/bin/env bash`

Also might I suggest installing `shellcheck` and if you use vscode the shellcheck extension. It will give you best practices and help spot errors in your bash code

https://www.shellcheck.net/

If you run homebrew it should be a `brew install shellcheck`, but it might be in the apt repos as well

Reply:
Thank you!